industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,6 @@
+1. Protein structure was prepared following the maestro instructions: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/pull/89. Afterward NMA cappings were
+converted to NME. I had to add the capping manually for PRO 554, and every chain terminus after.
+2.The structure did not contain any cofactors and no non natural amino acids were found,
+therefore no modifications.
+2. Ligands were left unchanged, no protonation or conformational states were detected.
```

Comment:
Could you update the ligand prep instructions to reflect that you only kept a single binding mode?